packages/ignite-basic-generators/templates/examples/GridExample.js.ejs

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React, { PropTypes } from 'react'
+import { View, Text, ListView } from 'react-native'
+import { connect } from 'react-redux'
+// import { Actions as NavigationActions } from 'react-native-router-flux'
```

Comment:
A lot of noise & garbage coming in here with these examples.  Let's clean these up in the future.